lib/matplotlib/transforms.py

```diff
@@ -517,14 +517,17 @@ def transformed(self, transform):
         Return a new :class:`Bbox` object, statically transformed by
         the given transform.
         """
-        return Bbox(transform.transform(self.get_points()))
+        pts = self.get_points()
```

Comment:
@dopplershift Just curious, should this fix have been applied to Transform.transform_bbox as well?  Right now we have two methods for transforming bboxes with subtly different semantics, it would seem better too only keep one of them...

Reply:
Quite possibly? My memory of doing this was that this was what it took to unbreak zooming (and maybe panning) on these skewed plots. It's possible that `transform_bbox` just isn't used in any code path I've hit.